src/systemctl/systemctl-logind.c

```diff
@@ -96,16 +96,16 @@ int logind_check_inhibitors(enum action a) {
         char **s;
         int r;
 
-        if (arg_ignore_inhibitors || arg_force > 0)
+        if (arg_check_inhibitors < 0 || arg_force > 0)
```

Comment:
hmm, this check should be == 0, no?

i.e. this should be what we typically call an "extended boolean", i.e. 

1. `== 0` is false/no/off,
2. `> 0` is true/yes/on,
3. and `< 0` is auto/default/dontknow.